com.cloudchewie.cloudotp.yaml

```diff
@@ -0,0 +1,157 @@
+app-id: com.cloudchewie.cloudotp
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.vala
+command: CloudOTP
+finish-args:
+  - --share=ipc
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=network
+  - --device=dri
+  - --talk-name=org.kde.StatusNotifierWatcher
+  - --talk-name=org.freedesktop.Notifications
```

Comment:
does this use libnotify for notifications?

Reply:
yes

Reply:
you can drop this then, libnotify uses the portal by default.

Reply:
done.